src/homework/tic_tac_toe/main.cpp

```diff
@@ -13,26 +13,63 @@ int main()
 	//Variable and class definitions.
 	Game game;
 	string player;
+	string again;
 	int position;
+	
 
-	//Prompts the user to enter in either 'X' or 'O' to being playing tic tac toe.
-	cout<<"Ready to play tic tac toe? To start, type either X or O to claim your marker: ";
-	cin>>player;
+	//While loop that keeps the game running. After the first game played, the user will be asked to input 'Y/y' in order to play again.
+	while(again == "Y" or 'y')
+	{
+		//Asks for the player to input their desired marker. 
+		cout<<"Welcome To Tic Tac Toe! Select Either X or O to pick your marker and get started! ";
+		cin>>player;
```

Comment:
Remove this code, loop prompts for input already, after using player in start game call set player= "" to reset it.